### PR TITLE
Support setting `priorityClassName` for memcached

### PR DESCRIPTION
**Pull request by bmcstdio**
This PR adds support for setting `priorityClassName` on the Flux and memcached pods.

### CODE REVIEW

File: chart/flux/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.18.0"
-version: 1.2.0
+version: 1.2.1
```
Comment (hiddeco) [inline chart/flux/Chart.yaml]: The release of our Helm chart is not automated, and the version is thus bumped during the release to not skip any versions.

File: chart/flux/README.md [hunk 2]
```diff
@@ -263,6 +263,7 @@ The following tables lists the configurable parameters of the Flux chart and the
 | `memcached.securityContext`                       | [See values.yaml](/chart/flux/values.yaml#L176-L179) | Container security context for memcached
 | `memcached.nodeSelector`                          | `{}`                                                 | Node Selector properties for the memcached deployment
 | `memcached.tolerations`                           | `[]`                                                 | Tolerations properties for the memcached deployment
+| `memcached.priorityClassName`                     | `""`                                                 | The name of the priority class to assign to the memcached pod(s).
```
Comment (hiddeco) [inline chart/flux/README.md]:
```suggestion
| `memcached.priorityClassName`                     | `""`                                                 | The name of the priority class to assign to the memcached pod.
```

As the replica count is a hardcoded `1` and the release strategy `Replace`.

File: chart/flux/values.yaml [hunk 3]
```diff
@@ -277,3 +278,5 @@ hostAliases: {}
 #    hostnames:
 #    - "foo.remote"
 #    - "bar.remote"
+
+priorityClassName: ""
```
Comment (hiddeco) [inline chart/flux/values.yaml]: Can you add a small note that this is the `priortiyClassName` for the Flux pod?

File: chart/flux/README.md [hunk 4]
```diff
@@ -273,6 +274,7 @@ The following tables lists the configurable parameters of the Flux chart and the
 | `syncGarbageCollection.dry`                       | `false`                                              | If enabled, fluxd won't delete any resources, but log the garbage collection output (see [garbage collection](/docs/references/garbagecollection.md))
 | `manifestGeneration`                              | `false`                                              | If enabled, fluxd will look for `.flux.yaml` and run Kustomize or other manifest generators
 | `hostAliases`                                     | `{}`                                                 | Additional hostAliases to add to the Flux pod(s). See <https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/>
+| `priorityClassName`                               | `""`                                                 | The name of the priority class to assign to the Flux pod(s).
```
Comment (hiddeco) [inline chart/flux/README.md]:
```suggestion
| `priorityClassName`                               | `""`                                                 | The name of the priority class to assign to the Flux pod.
```

As the replica count is a hardcoded `1` and the release strategy `Replace`.

**Comment by bmcstdio** [comment]
@hiddeco thank you for the review, and sorry for having taken so long to get back to this. I've done as you requests and added only the missing bits, while keeping the chart version unchanged.

File: chart/flux/README.md (hunk 4 above)
Comment (hiddeco) [reply chart/flux/README.md]: This entry can be removed from the table as it was already added in another PR.

Comment (bmcstdio) [reply chart/flux/README.md]: Done!

**Comment by hiddeco** [comment]
Thank you @bmcstdio :tulip:

**Pull request merged**